Client/src/components/pages/Dashboard/AdminDashboard/AdminUsers/index.jsx

```diff
@@ -77,39 +82,22 @@ function Index() {
                                     value={searchQuery}
                                     className="pl-8 pr-4 py-2 rounded border w-48"
                                 /> */}
-                                <Input placeholder='Search...' onChange={handleSearchChange} value={searchQuery} className="pl-8 pr-4 py-2 rounded border w-48"/>
+                                <Input placeholder='Search...' onChange={handleSearchChange} value={searchQuery} className="pl-8 pr-4 py-2 rounded border w-48" />
                                 <div className="absolute left-2 top-1/2 transform -translate-y-1/2 text-gray-400">
                                     <AiOutlineSearch />
                                 </div>
                             </div>
                             <Link to="/admin-createUser" className="inline-block px-4 py-2 bg-green-500 text-white rounded hover:bg-green-600 mr-5 ml-5">+ ADD</Link>
                         </div>
-                        
-                        <table className="w-full border-collapse">
-                            <thead>
-                                <tr>
-                                    <th className="border px-4 py-2">ID</th>
-                                    <th className="border px-4 py-2">Name</th>
-                                    <th className="border px-4 py-2">Email</th>
-                                    <th className="border px-4 py-2">Password</th>
-                                    <th className="border px-4 py-2">Actions</th>
-                                </tr>
-                            </thead>
-                            <tbody>
-                                {slicedData.map((d, i) => (
-                                    <tr key={i}>
-                                        <td className="border px-4 py-2">{d.id}</td>
-                                        <td className="border px-4 py-2">{d.name}</td>
-                                        <td className="border px-4 py-2">{d.email}</td>
-                                        <td className="border px-4 py-2">{d.password}</td>
-                                        <td className="border px-4 py-2 flex justify-center items-center space-x-2">
-                                            <Link to={`/admin-update/${d.id}`} className="inline-block px-2 py-1 bg-gray-500 text-white rounded hover:bg-gray-600 ml-2">Edit</Link>
-                                            <button className="inline-block px-2 py-1 bg-red-500 text-white rounded hover:bg-red-600 ml-2" onClick={() => handleDelete(d.id)}>Delete</button>
-                                        </td>
-                                    </tr>
-                                ))}
-                            </tbody>
-                        </table>
+
+                        <Table data={slicedData} headers={[
+                            { key: 'id', label: 'ID' },
```

Comment:
same as above

Reply:
Okay, Noted, Applying the changes and Re-pushing the code.